README.rst

```diff
@@ -42,21 +42,17 @@ Installation
 ============
 
 We recommend the `Anaconda Python distribution <https://www.anaconda.com/products/individual>`_.
-To install ``hnn-core``, you first need to install its dependencies::
+To install ``hnn-core``, simply do::
 
-	$ conda install numpy matplotlib scipy
+   $ pip install hnn_core
 
-Additionally, you would need Neuron which is available here:
-`https://neuron.yale.edu/neuron/ <https://neuron.yale.edu/neuron/>`_.
-Note installing Neuron using the traditional installer may make it hard to manage
-within virtual environments such as Anaconda. For this reason, we recommend the
-pip installer:
-
-	$ pip install NEURON
+and it will install ``hnn-core`` along with the dependencies which are not already installed.
```

Comment:
```suggestion
which will install ``hnn-core`` along with the dependencies that are not already installed. The most notable of these is NEURON `https://neuron.yale.edu/neuron/ <https://neuron.yale.edu/neuron/>`_.
```

Reply:
Thought we should mention NEURON on the start page (or is it just "Neuron"?). EDIT: I see we have the link below in a different context... Might be nice to acknowledge here too, though?

Reply:
I'm hesitant to provide the link because users may go for the traditional install if they open the website and that will cause us unnecessary headaches

Reply:
> 
> 
> Thought we should mention NEURON on the start page (or is it just "Neuron"?). 

I'm a fan of NEURON to avoid confusion with the journal.